assignments/callbacks.js

```diff
@@ -27,24 +27,39 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+    return cb(arr.length);
 }
+getLength(items, len => console.log(len));
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
+last(items, lastItem => console.log(lastItem));
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+    return cb(x+y);
 }
+sumNums(1,2,ans => console.log(ans));
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+    return cb(x*y);
 }
+multiplyNums(5,6,ans => console.log(ans));
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  for (let i=0; i<list.length; i++){
```

Comment:
It is exemplary that you implement this yourself, but in the future you can use the `includes` array method